content/pages/disability-benefits/claims-appeal/request-a-priority-review.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page-breadcrumbs.html
+title: Request a Priority Review
+display_title: Request a Priority Review
+order: 1
+plainlanguage: 11-02-16 certified in compliance with the Plain Writing Act
+template: detail-page
+---
+<div itemscope itemtype ="http://schema.org/HowTo">
```

Comment:
Is there an extra space before `=`?

Reply:
Copy pasta. Good eye. :female_detective: